docs/06_how-to-guides/50_how-to-create-and-use-action-wrappers.md

```diff
@@ -3,7 +3,19 @@ content_title: How to create and use action wrappers
 link_text: "How to create and use action wrappers"
 ---
 
-1. Start with a contract `multi_index_example` which has an action `mod` defined like below in file `multi_index_example.hpp`; the action modifies the integer value `n` stored for row with key `user`.
+## Overview
+
+This how-to guide provides instructions on how to create and use an action wrapper in a smart contract.
```

Comment:
perhaps 

"This guide provides instructions on how to create and use an action wrapper in a smart contract." to avoid repetitive use of how to

Reply:
I have used

## Overview
This guide provides instructions on how to buy RAM for an EOSIO blockchain account using the cleos CLI tool. RAM is a system resource and is used to store smart contract state and account information.

The example uses cleos CLI tool to buy RAM for the alice account. The alice account pays for the RAM and the alice@active permisssion authorizes the transaction.  [first draft]

Reply:
do we need the example section, explaining what this how to explicitly shows?

Reply:
I do not understand the last question. This how to it is showing example of code...

Reply:
In the meetings it was discussed that the overview should be generic, then describe what the how to actually shows.

Reply:
in my example:

This is a generic overview:
This guide provides instructions on how to buy RAM for an EOSIO blockchain account using the cleos CLI tool. RAM is a system resource and is used to store smart contract state and account information.

This states specifically what this how to shows:
The example uses cleos CLI tool to buy RAM for the alice account. The alice account pays for the RAM and the alice@active permisssion authorizes the transaction. [first draft]

The explicit question is "Do you need to add a statement whcih states specifcally what this how to shows"

Hope that helps

Reply:
yes yes I like it.